yb-voyager/cmd/common.go

```diff
@@ -55,8 +55,16 @@ import (
 var (
 	metaDB               *metadb.MetaDB
 	PARENT_COMMAND_USAGE = "Parent command. Refer to the sub-commands for usage help."
+	StartTime            time.Time
 )
 
+func PrintElapsedDuration() {
+	fmt.Printf("End time: %s\n", time.Now())
```

Comment:
can we use `utils.PrintAndLog` instead of `Printf`

Reply:
Done.